src/main/java/uk/nhs/digital/iucds/middleware/MiddlewareSchedulerTask.java

```diff
@@ -94,68 +96,24 @@ public void sendMails() throws InterruptedException {
 
       while (findResults.getTotalCount() > 0) {
         for (Object item : findResults.getItems()) {
           try {
             EmailMessage emailMessage = (EmailMessage) item;
-            emailMessage.load(
-                new PropertySet(BasePropertySet.FirstClassProperties, ItemSchema.MimeContent));
-            log.info("attachment count: {} ", emailMessage.getAttachments().getCount());
-            Attachment attachment = emailMessage.getAttachments().getItems().get(0);
-            stopwatch.finishStage("Getting html attchement from email");
-            if (attachment instanceof FileAttachment) {
-              FileAttachment fileAttachment = (FileAttachment) attachment;
-              fileAttachment.load();
-
-              if (fileAttachment.getContentType().equalsIgnoreCase(MimeTypeUtils.TEXT_HTML_VALUE)) {
-                // convert bytes[] to string
-                String htmlString = new String(fileAttachment.getContent(), StandardCharsets.UTF_8);
-                Document doc = Jsoup.parse(htmlString);
-                byte[] transform = new PDFTransformer().transform(doc.html());
-                stopwatch.finishStage("pdf transformation");
-                
-                client.sendMDM(transform);
-                stopwatch.finishStage("Sent MDM");
-
-                // Create an email message and set properties on the message.
-                EmailMessage message = new EmailMessage(service);
-                message.setSubject(getParameter("EMS_REPORT_SUBJECT"));
-                message.setBody(new MessageBody(getParameter("EMS_REPORT_BODY")));
-                message.getToRecipients().add(getParameter("EMS_REPORT_RECIPIENT"));
-                FileAttachment addFileAttachment =
-                    message.getAttachments().addFileAttachment(createFileName(doc), transform);
-                addFileAttachment.setContentType("application/pdf");
-                stopwatch.finishStage("Added pdf attchement to mail");
-                message.send();
-                stopwatch.finishStage("Sent an email with pdf attachement");
-              }
-
-            } else if (attachment instanceof ItemAttachment) {
-              ItemAttachment itemAttachment = (ItemAttachment) attachment;
-
-              String name = itemAttachment.getName();
-              log.info("ItemAttachment name : {} ", name);
-              Item attachedItem = itemAttachment.getItem();
-              log.info("ItemAttachment attachedItem : {} ", attachedItem);
-            }
-            emailMessage.setIsRead(true);
-            emailMessage.update(ConflictResolutionMode.AlwaysOverwrite);
-            stopwatch.finishStage("Making email unread after reading email");
+            
+            Attachment attachmentFromEmailMessage = getAttachmentFromEmailMessage(emailMessage);
+            
+            getFileContentFromAttachment(attachmentFromEmailMessage);
+            
+            setMailsIsRead(emailMessage);
```

Comment:
we need to either delete the email or have another periodic job running and deleting the read email after eg. 7 days. otherwise the mailbox might hit the limit for size